lessonbuilder/tool/src/resources/messages_en_GB.properties

```diff
@@ -1,9 +1,6 @@
 # Gradebook
-simplepage.comments-graded=Create Markbook entries for these
-simplepage.gbname-expected=Markbook item name must be supplied.
-simplepage.no-gradebook=This item couldn't be marked for grading. The most likely issue is that no Markbook tool has been added to this site yet.
-simplepage.page.gradebook=Create Markbook entry when page is completed.
-simplepage.question-gradebook-title=Title in Markbook:
+simplepage.comments-graded=Create Gradebook entries for these
+simplepage.page.gradebook=Create Gradebook entry when page is completed.
```

Comment:
This line can go I think as it doesn't reference marks.

Reply:
The default entry is 'Create Gradebook item when page is completed.' so I made the change above.

Reply:
Ok